mainnet/0.74-proposal-templates/README.md

```diff
@@ -125,3 +125,12 @@ Therefore it is recommended a new trigger which results in a long auction for la
 - `horizon` set to `21600`, `probability` set to `0.9999999`, and `auctionExtension` set to `86400` (or `1d`).
 
 Note: For the ETH/USDT market in particular, this means the addition of two new price monitoring triggers as it currently only uses the single trigger.
+
+### Liquidity SLA & risk model changes
+
+The template also contains changes to the following parameters that were proposed by a community member.  Since these were communnity driven the ratioinale is not discussed here and can be found on the forums..
+
+- `liquiditySlaParameters.commitmentMinTimeFraction`: `0.75`
+- `liquiditySlaParameters.performanceHysteresisEpochs`: `0`
+- `liquiditySlaParameters.slaCompetitionFactor`: `0.8`
+- `logNormal.params.sigma`: `1.0`
```

Comment:
BTC/USDT is 1.5 in: https://github.com/vegaprotocol/governance-templates/pull/21